### DISCUSSION TITLE
watch the discussions

### DISCUSSION THREAD

**Discussion by jmarca** [community/community#3550] [Discussions] [answered]
Hi,  I was wondering if there is a way to turn on watching in a repo for discussions only.  That is, I don't care about all the other noise related to issues, pull requests, and commits.  I only want to get new discussions bounced to my email.

**Comment by laeubi** [comment, marked as answer]
You can choose `Watch > Custom > Discussions` for a github repository

**Comment by jmarca** [comment]
Cool thanks. That is exactly what I was looking for.
